src/System.Private.ServiceModel/tests/Scenarios/Bridge/Bridge/Bridge.csproj

```diff
@@ -0,0 +1,106 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{1E08A11A-A5E7-40F0-8CAE-4797259970F3}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Bridge</RootNamespace>
+    <AssemblyName>Bridge</AssemblyName>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
+    <OutputPath>..\bin\</OutputPath>
+    <RestorePackages>true</RestorePackages>
+    <PackageOutputDir>..\..\..\..\..\packages</PackageOutputDir>
```

Comment:
Regarding OutputPath, we need a convention for where our tools binaries land so they are not inside the enlistment.  One possibility would be create a new targets file at the root (imported by override.targets) that sets some properties to define this This would also be an ideal place to set $(BridgeResourceFolder) so that we can have all resources build their outputs to the same known location where the Bridge looks.